src/express.re

```diff
@@ -36,6 +36,22 @@ module Request = {
   external baseUrl: t => string = "" [@@bs.get];
   /** [baseUrl request] returns the 'baseUrl' property */
 
+  external cookies : t => option (Js.Dict.t Js.Json.t) = "" 
```

Comment:
Yes it is possible but it won't add any sort of guarantee that the cookies are not 'null' since the user might not have called make. Also the reason i kept it in the `Request` module is to make the interface very close to the JS Api.